Misc/NEWS.d/next/Library/2023-06-21-04-31-20.gh-issue-0.FAVXRp.rst

```diff
@@ -0,0 +1,8 @@
+I ran into a bug while using OpenCV, Flask, and Japanese fonts through Pillow. Here's the pertinent section of the traceback:
+
+\AppData\Local\Programs\Python\Python311\Lib\http\server.py", line 513, in send_response_only
+    (self.protocol_version, code, message)).encode(
+                                            ^^^^^^^
+UnicodeEncodeError: 'latin-1' codec can't encode character '\u3007' in position 120: ordinal not in range(256)
+
+I simply changed latin-1 in two spots to utf-8, and the problem seems to be solved.
```

Comment:
Could you create an issue with the text above but slightly adjusted like below, please?

<pre>
I ran into a bug while using OpenCV, Flask, and Japanese fonts through Pillow. Here's the pertinent section of the traceback:

```plain
\AppData\Local\Programs\Python\Python311\Lib\http\server.py", line 513, in send_response_only
    (self.protocol_version, code, message)).encode(
                                            ^^^^^^^
UnicodeEncodeError: 'latin-1' codec can't encode character '\u3007' in position 120: ordinal not in range(256)
```

The problem can be solved by changing `latin-1` in two spots to `utf-8`.
</pre>

Reply:
I'll get right on it.

Reply:
@fkkjohn No, I've bound this PR to it so we will close it after everything is merged.

By the way, we can simplify this file (because `Misc/NEWS.d` directory is compiled into a changelog); I've done this on your behalf. If necessary, you can pull the changes into your local copy.

Reply:
> Should I close the issue as well?

@fkkjohn No, I've bound this PR to it so we will close it after everything is merged.

Reply:
By the way, we can simplify this file (because `Misc/NEWS.d` directory is compiled into a changelog); I've already done this on your behalf. If necessary, you can pull the changes into your local copy.

Reply:
> Should I close the issue as well?

No need to do this, I've bound this PR to the issue so we will close it after everything is merged.

Reply:
By the way, we can simplify this file (because `Misc/NEWS.d` directory is compiled into a changelog); I've already done this on your behalf. If necessary, you can pull the changes into your local copy.

Reply:
Thank you. I appreciate the help. I'm not well versed at github yet, and the Python repository is understandably more demanding than smaller projects.